package.json

```diff
@@ -133,6 +131,7 @@
     "@restart/ui": "^1.0.1",
     "@tailwindcss/forms": "^0.5.0",
     "@yext/analytics": "^0.2.0-beta.3",
+    "@yext/search-headless-react": "^1.3.0",
```

Comment:
why was this changed to a regular dependency instead of dev and peer dependency?

Reply:
not sure how that happened, I'll take `package.json` from the feature branch and rename it again